runtime/expr/cutter.go

```diff
@@ -9,15 +9,13 @@ import (
 )
 
 type Cutter struct {
-	zctx        *zed.Context
-	fieldRefs   field.List
-	fieldExprs  []Evaluator
-	lvals       []*Lval
-	outTypes    *zed.TypeVectorTable
```

Comment:
I think this can stay here and doesn't need to be in recordBuilderCachedTypes.

Reply:
(This being `outTypes`.)

Reply:
yep